src/conv.jl

```diff
@@ -329,6 +329,19 @@ for conv in [:conv, :depthwiseconv]
     end
 end
 
+function rrule(::typeof(∇conv_filter), x, dy, cdims; kw...)
+    function ∇conv_filter_pullback(Δ)
+        Δ = colmajor(Δ)
+        return (
+            NoTangent(),
+            @thunk(∇conv_data(dy, unthunk(Δ), cdims, kw...)),
+            @thunk(conv(x, unthunk(Δ), cdims, kw...)),
```

Comment:
I think this should only unthunk once (as that might be the expensive bit):
```suggestion
        Δ1 = colmajor(unthunk(Δ))
        return (
            NoTangent(),
            @thunk(∇conv_data(dy, Δ1, cdims, kw...)),
            @thunk(conv(x, Δ1, cdims, kw...)),
```